groups/hash-check/main.tf

```diff
@@ -0,0 +1,40 @@
+
+provider "aws" {
+  region  = var.region
+  version = "~> 2.50.0"
+}
+
+provider "vault" {
+  auth_login {
+    path = "auth/userpass/login/${var.vault_username}"
+
+    parameters = {
+      password = var.vault_password
+    }
+  }
+}
+
+terraform {
+  backend "s3" {
+  }
+}
+
+data "aws_caller_identity" "current" {}
+
+module "hash_check" {
+  source = "./hash-check"
+
+  aws_account                           = var.aws_account
+  aws_account_id                        = data.aws_caller_identity.current.account_id
+  aws_profile                           = var.aws_profile
```

Comment:
The three `aws_*` variables here aren't referenced in the Terraform module code so are currently redundant and could be removed:

```suggestion
```

Reply:
Should we leave the variables in the variables.tf file similar to the region?